src/App.jsx

```diff
@@ -1,12 +1,145 @@
 import './App.css';
+import axios from 'axios';
+import {useState} from 'react';
+import getContinentColor from "./helpers/getRegionColour.js";
+import worldMap from "./assets/world_map.png"
+import population from "./helpers/population.js";
 
 function App() {
+    const [countries, setCountries] = useState([]);
+
+    const [error, toggleError] = useState(false);
+
+    const [countryInfo, setCountryInfo] = useState({});
+
+    const [searchQuery, setSearchQuery] = useState('');
+
+        async function fetchWorldinfo() {
+        toggleError(false);
+        try {
+            const result = await axios.get('https://restcountries.com/v3.1/all');
+            console.log(result.data);
+
+            const sortedPop = result.data.sort((a, b) => a.population - b.population);
+            setCountries(sortedPop);
+        } catch (e) {
+            console.error(e);
+            toggleError(true);
+        }
+    }
+
+    async function fetchCountryInfo(countryName) {
+        toggleError(false);
+        try {
+            const result = await axios.get(`https://restcountries.com/v3.1/name/${countryName}`);
+            console.log(result.data);
+            const country = result.data[0];
+            console.log("Name:", country);
+            const countryCapital = result.data[0].capital
+            console.log("Capital:", countryCapital);
+            setCountryInfo(country);
+
+        } catch (e) {
+            console.error(e);
+            toggleError(true);
+        }
+
+    }
+    const handleSearch = (event) => {
+        if (event.key === "Enter" || event.type === "click") {
+            if (searchQuery) {
+                fetchCountryInfo(searchQuery);
+                setSearchQuery("");
+            } else {
+                toggleError(true);
+            }
+        }
+    };
 
     return (
         <>
-            Maak je applicatie hier!
+            <header>
+                <img
+                    src={worldMap}
+                    alt="World map"
+                    style={{width: "1000px"}}
+                />
+            </header>
+            <body>
```

Comment:
Het body element staat al in de index.html file die de basis vormt voor de applicatie. Je gebruikt deze dus nooit als semantisch element tussen andere semantische elementen. De basis dient altijd te bestaan uit header - main - footer.